src/chain/network_event.rs

```diff
@@ -61,6 +70,29 @@ impl EventSigPayload {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
+pub struct RealBlsEventSigPayload {
```

Comment:
I guess "Real" will be removed once the work is complete?

Reply:
Yes, it is just a bridge so some pieces can co-exist while we are solving the missing pieces.